src/EditorFeatures/CSharpTest/SignatureHelp/ConstructorInitializerSignatureHelpProviderTests.cs

```diff
@@ -265,12 +266,13 @@ class D
 {
     D() [|: this(i: null$$|]) { }
 
-    D(D filtered) => throw null;
+    D(D other) => throw null;
```

Comment:
i would prefer we not change the input state of existing tests.  instead, just show what chnaged in teh behavior.  then, add a new test with this new input state.